docs/how-does-it-work/phases-of-metagame.md

```diff
@@ -16,7 +16,7 @@ The meme of this phase is _A Decentralized Factory_; a collective of decentraliz
 
 ### Goals:
 
-- Build a community of people strongly aligned with the _MetaManifesto_.
+- Build a community of people strongly aligned with the [_MetaManifesto_](http://localhost:3000/docs/wtf-is-metagame/metafam-way).
```

Comment:
Hey @jonathanprozzi I notice you've got a link to localhost here. 😅

Reply:
Otherwise, this is good to go.

Reply:
My bad!! thanks for catching that :)